deployments/container/Dockerfile

```diff
@@ -34,14 +34,35 @@ ARG GOLANG_VERSION=x.x.x
 ARG BUILDARCH
 ARG TARGETARCH
 
+# Install dependencies for Go build. Do not clear apt cache (does not
+# leak into prod stage).
 RUN apt-get update && \
     apt-get install -y \
         wget \
         make \
         git \
         gcc-aarch64-linux-gnu \
-        gcc && \
-    rm -rf /var/lib/apt/lists/*
+        gcc
+
+# Install dependencies for `bash-static` build.
+RUN apt-get install -y gpg curl autoconf
+
+# Build static bash binary (against musl).
+WORKDIR /bashbuild
+RUN git clone https://github.com/robxu9/bash-static/
+RUN ARCH="$TARGETARCH" && \
+    [ "$ARCH" = "arm64" ] && ARCH="aarch64" || true && \
+    [ "$ARCH" = "amd64" ] && ARCH="x86_64" || true && \
+    echo "detected arch: $ARCH" && \
+    cd bash-static && git checkout 021f5f29f665c92ca16a369d9f27e288c3aed0c6 && \
+    sed -i 's|https://ftp\.gnu\.org/gnu|https://ftpmirror.gnu.org/|g' ./build.sh && \
+    sed -i 's/-sLO/-sSfLO --retry 300 --connect-timeout 20 --retry-delay 2/g' ./build.sh && \
+    bash version-52.sh && ./build.sh linux $ARCH
+
+# With above's commit, this emits
+# 'GNU bash, version 5.2.37(1)-release (aarch64-unknown-linux-musl)'
+RUN cd /bashbuild/bash-static/releases && ./bash-5.2-static --version
+RUN mv /bashbuild/bash-static/releases/bash-5.2-static /bashbuild/bash
```

Comment:
Can the `bash-5.2-static` also somehow be parameterized (or maybe we search for bash-*-static and execute that)?

Reply:
Bash 5.2.x has been going strong for ~three years (we're at 5.2.37 now). Only a few days ago, 5.3.0 was released, and I doubt we want to transition to that any time soon (if ever).

But yes. we can do that. :)